acceptance/setup/pre_suite/90_install_devel_puppetdb.rb

```diff
@@ -10,6 +10,8 @@
       if (test_config[:database] == :postgres)
         install_postgres(database)
       end
+      # TODO: fix me once merged and ezbake 0.3.8 is released
```

Comment:
This needs removal once the other PR over in ezbake is merged/released etc: https://github.com/puppetlabs/ezbake/pull/250